dags/vds_pull.py

```diff
@@ -0,0 +1,131 @@
+from os import path
+import sys
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.operators.python import PythonOperator
+from airflow.hooks.base import BaseHook
+from airflow.providers.slack.operators.slack_webhook import SlackWebhookOperator
+from airflow.providers.postgres.hooks.postgres import PostgresHook
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+from airflow.models import Variable
+
+#CONNECT TO ITS_CENTRAL
+itsc_bot = PostgresHook("itsc_postgres")
+
+#CONNECT TO BIGDATA
+vds_bot = PostgresHook("vds_bot")
+
+try:
+    repo_path = '/home/gwolofs/bdit_data-sources'
```

Comment:
This shouldn't be hard-coded or pointing to personal home folders. Check this code snippet as an example of a better implementation https://github.com/Toronto-Big-Data-Innovation-Team/vfh_data_mgmt/blob/bab9e43bda42215b6193f2b710f8cade1eb37e06/dags/unit_validation.py#L23